public/viewtopic.php

```diff
@@ -393,4 +387,3 @@
 
 </body>
 </html>
-
```

Comment:
a minor thing https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
yeah, I can see the problems with it when I have to deal with text files on my shell scripts.
But the previous version of this file had two new lines in the end, now it has only one. ;)